code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1,17 +1,8 @@
+#define ALCOHOL_RATE 0.015 //The rate at which alcohol affects you; by default, one glass of whiskey gets you just above tipsy
```

Comment:
gets you just above what? drunk level? shitfaced level?

Also, are you sure about your calculations? With whiskey I get: 0.015 * 75 = 1.125, minus the 0.2 drunkenness reduction per tick, you get +0.925 per tick of whiskey consumed. Since a glass is 50u and you metabolize 0.4u per tick, a full glass of whiskey makes you reach drunkenness = 115